streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/SubscriptionInfoTest.java

```diff
@@ -246,7 +233,7 @@ public void generatedVersion3And4ShouldBeDecodableByLegacyLogic() {
     }
 
     @Test
-    public void generatedVersion3And4ShouldDecodeLegacyFormat() {
+    public void generatedVersion3To5ShouldDecodeLegacyFormat() {
```

Comment:
We didn't change the method name when we added version 5. Since we're about to add a non-legacy version 6, this method won't get renamed again.